src/pages/en/developers/apis/6-terms-of-use/index.md

```diff
@@ -2,6 +2,74 @@
 title: Terms of use
 ---
 
-All data _(except for ODbL licensed data listed below)_ in these APIs is licensed under the [Creative Commons BY 4.0 International](https://creativecommons.org/licenses/by/4.0/) -license. When using the data, please add an attribution where you mention the licensee and the year when the data was retrieved (e.g. © Helsinki Region Transport 2019).
+## TERMS OF USE FOR DIGITRANSIT API REGISTERED USAGE
 
-Route geometries from routing API, address data from geocoding API and map data from map API are licensed under the [Open Database License (ODbL)](https://opendatacommons.org/licenses/odbl/index.html).
+### 0. Intro on Digitransit  
+
+Digitransit Platform is an open-source journey planning solution that combines open-source components into a route planning service. It has Mobile friendly user interface, Map tile serving, Geocoding, and data conversion tools. Route planning algorithms and APIs are provided by Open Trip Planner (OTP). More on Digitransit, please visit: https://digitransit.fi/en/
```

Comment:
Digitransit Platform is an open-source journey planning solution that combines open-source components into a route planning service. It has Mobile friendly user interface, Map tile serving, Geocoding, and data conversion tools. Route planning algorithms and APIs are provided by Open Trip Planner (OTP).  For more information about Digitransit, please visit: https://digitransit.fi/en/